scripts/Convert-ProwlarrSupportedIndexersToMarkdownTable.ps1

```diff
@@ -0,0 +1,280 @@
+    .NOTES
+        This script has been tested on Windows PowerShell 7.1.3
+    .EXAMPLE
+    PS> .\Convert-ProwlarrSupportedIndexersToMarkdownTable.ps1 -Commit 1.1.1.1 -Build "test" -AppAPIKey "asjdhfjashdf89787asdfsad87676" -AppBaseURL http://prowlarr:9696 -OutputFile "supported-indexers.md"
+    .EXAMPLE
+    PS> .\Convert-ProwlarrSupportedIndexersToMarkdownTable.ps1 -Commit 1.1.1.1 -Build "test" -AppAPIKey "asjdhfjashdf89787asdfsad87676" -AppBaseURL http://prowlarr:9696
+    .EXAMPLE
+    PS> .\Convert-ProwlarrSupportedIndexersToMarkdownTable.ps1 -Commit 1.1.1.1 -Build "test" -AppAPIKey "asjdhfjashdf89787asdfsad87676" -OutputFile "supported-indexers.md"
+    .EXAMPLE
+    PS> .\Convert-ProwlarrSupportedIndexersToMarkdownTable.ps1 -Commit 1.1.1.1 -AppAPIKey "asjdhfjashdf89787asdfsad87676"
+#>
+
+[CmdletBinding()]
+param (
+    [Parameter(Position = 1)]
+    [string]$Commit,
+    [Parameter(Position = 2)]
+    [string]$Build,
+    [Parameter(Mandatory, Position = 3)]
+    [string]$AppAPIKey,
+    [Parameter(Position = 4)]
+    [System.IO.FileInfo]$OutputFile = ".$([System.IO.Path]::DirectorySeparatorChar)supported-indexers.md",
+    [Parameter(Position = 5)]
+    [uri]$AppBaseURL = 'http://localhost:9696'
+),
+
+# Gather Inputs & Variables
+## User Inputs
+### Convert Params to match vars
+$app_baseUrl = $AppBaseURL
+$app_apikey = $AppAPIKey
+## Start Variables
+### Application Details
+$app_api_version = 'v1'
+$app_api_path = '/api/'
+$app_api_endpoint_version = '/system/status'
+$app_api_endpoint_indexer = '/indexer/schema'
+$headers = @{'X-Api-Key' = $app_apikey }
+### Github App Info
+$gh_app_org = 'Prowlarr'
+$gh_app_repo = 'Prowlarr'
+### Wiki Details
+$wiki_link = 'https://wiki.servarr.com'
+$wiki_app_path = '/prowlarr'
+$wiki_page = 'supported-indexers'
+$wiki_bookmark = '#'
+### Page Formatting
+$markdown_escape_regex = '(\w)(\.|\[|\])(\w)'
+$markdown_escape_regex_rep = '$1\$2$3'
+$wiki_1newline = "`r`n"
+$wiki_2newline = "`r`n`r`n"
+$wiki_encoding = 'utf8'
+### Github Details
+$gh_web = 'https://github.com'
+$gh_web_commit = 'commit/'
+### Prowlarr Search String Dictionary
+$SearchTypes = @{
+    'Q'         = 'Text Query'
+    'Album'     = 'Album'
+    'Artist'    = 'Artist'
+    'Author'    = 'Author'
+    'Ep'        = 'Episode'
+    'Genre'     = 'Genre'
+    'ImdbId'    = 'IMDb ID'
+    'ImdbTitle' = 'IMDb Title'
+    'ImdbYear'  = 'IMDb Year'
+    'Label'     = 'Label'
+    'RId'       = 'TV Rage Id'
+    'Season'    = 'Season'
+    'Title'     = 'Title'
+    'TmdbId'    = 'TMDb Id'
+    'TraktId'   = 'Trakt Id'
+    'TvdbId'    = 'TVDb Id'
+    'TvMazeId'  = 'TV Maze Id'
+    'Year'      = 'Year'
+}
+
+function Invoke-SearchReplace
+{
+    param (
+        [Parameter(Mandatory)]
+        [PSCustomObject]
+        $SearchValue
+    )
+    if ($SearchValue)
+    {
+        foreach ($dict in $SearchTypes)
```

Comment:
You should really pass $searchTypes in as a parameter (scoping isn't generally an issue but it's tidier in case scoping has a change)